runtime/router.go

```diff
@@ -205,6 +211,11 @@ func NewRouter(gateway *Gateway) *Router {
 }
 
 func (router *Router) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	err := router.logRequest(r)
```

Comment:
This will fix an outstanding bad thing in RTAPI because we hook in early enough to log all 404 and 405.